src/Microsoft.DotNet.Wpf/specs/tooltip.md

```diff
@@ -0,0 +1,305 @@
+
+[FD] says "Once a tooltip is dismissed, it should not re-appear on subsequent hovers", but does not define "subsequent hover".
+Raising that question on [ET] produced no direct response, but rather the assertion that [FD] is wrong.
+Thus this spec proscribes no changes.
+WPF maintains the 4.8 behavior, which applies the rule only to tooltips dismissed by timeout;
+this becomes moot with the change to infinite timeouts.
+
+[FD] says "User tabs to an element with a tooltip.  The tooltip appears on focus."
+This spec extends "user tabs" to any keyboard navigation, so that arrow keys, page-up, et al. have the same effect as tab.
+This behavior was already added in 4.8.
+
+[FD] says (or rather shows) that Tab and Ctrl-Shift-F10 both open the tooltip.
+[ET] clarifies that "Tab" here means "tab-navigation to the parent element", rather than "typing TAB while the parent element has keyboard focus".
+
+On the same subject, [ET] also says that the two keyboard actions should be labeled as optional alternatives for meeting the [WCAG] requirement.
+The tab-navigation rule is intended for tooltips that follow the [FD] guidelines for tooltip usage and content, but apps that put lengthy or redundant text in tooltips may find the rule "too distracting".
+They can use the Ctrl-Shift-F10 rule instead, to avoid the distraction while still giving keyboard-only users a way to open the tooltip.
+Several legacy apps take advantage of this option, notably Office.
+
+A framework like WPF should leave this option up to the app,
+mirroring the app's choice of tooltip usage and content.
+WPF 4.8 mistakenly baked in the option:  the tab-navigation rule applies unless the tooltip derives from RibbonToolTip. 
+The ``ShowsToolTipOnKeyboardFocus`` properties return the option to the app.
+Unfortunately this only helps in .NET 6.0, as servicing updates to older versions cannot include new public API.
+
+[FD] says (shows) that Ctrl closes the tooltip.
+[ET] discussed what that really means - whether to act on KeyDown or KeyUp, whether combinations should cause both the combination action and dismiss the tooltip (e.g. should Ctrl+C both copy text and dismiss the tooltip), etc.
+The interpretation given here is the consensus.
+
+
+# Summary of behavior changes
+
+For convenience, here are the behavior changes described in detail above.
+
+1. Change default value for `ToolTipService.ShowDuration` property from 5000 to `Int32.MaxValue`.
+2. Moving the mouse within the buffer zone does not close the tooltip.
+3. (6.0 only) Properties `ToolTip.ShowsToolTipOnKeyboardFocus` and `TooltipService.ShowsToolTipOnKeyboardFocus` control whether acquiring keyboard focus shows the tooltip.
+4. Ctrl closes the tooltip.
+
+Also, some changes were previously made in .NET 4.8 (and appear in .NET Core 3.0, 3.1, and .NET 5.0), without documentation:
+
+1. Keyboard focus opens the tooltip.  (Except for RibbonToolTip.)
+2. Ctrl-Shift-F10 opens or closes the tooltip.
+
+# API Details
```

Comment:
Could you put this above the implementation section?

Reply:
I'd rather not.  This is a summary section, repeating material that appears earlier.  Summaries belong at the end, or at least after the things they summarize.